examples/confluent-server/docker-compose.yml

```diff
@@ -1,94 +1,76 @@
 ---
-version: '2'
 services:
-  zookeeper:
-    image: confluentinc/cp-zookeeper:latest
+  controller-1:
```

Comment:
Should you add 3 controllers to make the example more realisitic?

Reply:
good idea - I changed it to have 3 controllers